latest/src/node_activation.rs

```diff
@@ -202,7 +224,11 @@ pub trait ContractStakeModule {
         // send unstake command to Auction SC
         let auction_contract_addr = self.settings().get_auction_contract_address();
         let auction_contract = contract_proxy!(self, &auction_contract_addr, Auction);
-        auction_contract.unStake(node_ids, bls_keys.into());
+        if unstake_tokens {
```

Comment:
do not really like this if.

Reply:
yeah, me neither, but I didn't want to duplicate a lot of code, or change the model. Will refactor in the updated version.